Sources/CoreCommands/Options.swift

```diff
@@ -598,28 +599,30 @@ public struct TestLibraryOptions: ParsableArguments {
           help: .private)
     public var explicitlyEnableExperimentalSwiftTestingLibrarySupport: Bool?
 
-    private func isEnabled(_ library: TestingLibrary, `default`: Bool) -> Bool {
+    private func isEnabled(_ library: TestingLibrary, `default`: Bool, swiftCommandState: SwiftCommandState) -> Bool {
```

Comment:
I think that `state` might be a better label than `swiftCommandState`. The latter doesn't really tell me anything more, and this is a private function in `CoreCommands`.

Reply:
This label is consistent with all other uses of this type in argument position.